testsuite/forge.py

```diff
@@ -1111,6 +1124,11 @@ def test(
     assert forge_image_tag is not None, "Forge image tag is required"
     assert upgrade_image_tag is not None, "Upgrade image tag is required"
 
+    print("Using the following image tags:")
```

Comment:
should we add some of this in the report message?

Reply:
The comment output will print the image tag the test used, and in the case of compat, the test itself injects test report text regarding the order of upgrades as they roll out to the network